### PR TITLE
add Coin M0 files

### PR DESCRIPTION
**Pull request by bradanlane**
The Coin M0 is a simple SAMD21 board the size of a large coin. It has just enough build-in features for a desktop macropad for visual (neopixels) and/or audio (speaker) feedback. The 4MB of flash is plenty for code or files.

The images use a white background. If this is a problem, I will take new photos with a charcoal or black background.

### CODE REVIEW

File: _board/bradanlanestudio_coin_m0.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+layout: download
+board_id: "bradanlanestudio_coin_m0"
+title: "Bradán Lane STUDIO Coin M0 Download"
+name: "Bradán Lane STUDIO Coin M0"
+manufacturer: "Bradán Lane STUDIO"
+board_url: https://aosc.cc/eccn2025
+board_image: "bradanlanestudio_coin_m0.jpg"
+date_added: 2024-11-22
+family: atmel-samd
+features:
```
Comment (FoamyGuy) [inline _board/bradanlanestudio_coin_m0.md:11]: `USB-C` can be added here as a feature, that will make the filter on the downloads page find it.

**Review by FoamyGuy — CHANGES_REQUESTED** [review]
Lets add the USB-C feature to it. Looks good to me otherwise.

**Comment by bradanlane** [comment]
Nice catch. I've added "USB-C" to the features section.

**Review by FoamyGuy — APPROVED** [review]
Thank you!

**Pull request merged**